docs/design/lexical_conventions/operators.md

```diff
@@ -0,0 +1,66 @@
+# Operators
+
+<!--
+Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+Exceptions. See /LICENSE for license information.
+SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+-->
+
+<!-- toc -->
+
+## Table of contents
+
+-   [Overview](#overview)
+-   [Symbolic tokens](#symbolic-tokens)
+    -   [Details](#details)
+    -   [List](#raw-string-literals)
+-   [Alternatives considered](#alternatives-considered)
+-   [References](#references)
+
+<!-- tocstop -->
+
+## Overview
+
+Carbon has a fixed set of tokens that represent operators, defined by the language specification. Developers cannot define new tokens to represent new operators - however, there may be facilities to overload operators in the future.
+
+There are two kinds of tokens that represent operators: Symbolic tokens and keywords (for keyword operators, see [words](words.md)).
```

Comment:
I suspect this file should focus solely on symbolic tokens, rather than operators broadly. https://github.com/carbon-language/carbon-lang/blob/trunk/docs/design/expressions/README.md#operators lists operators, and note some operators are words (`and`, `or`, `not`, `as`). Further, some tokens aren't what we'd call operators (`{}` for example).

I think the operators might be slightly higher level than the lexical tokens, so just removing this section about operators and making the next the overview for this file may be better. Note that will necessarily change some of the phrasing, e.g. "Not all uses of symbolic tokens within the Carbon grammar will be as operators" might be better phrased as "Some, but not all, symbolic tokens are [operators](https://github.com/carbon-language/carbon-lang/blob/trunk/docs/design/expressions/README.md#operators)".